src/components/ItemsList.js

```diff
@@ -73,6 +73,24 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
+  const deleteItemFromShoppingList = (itemName) => {
+    const deleteItem = window.confirm(
+      `Are you sure you want to delete ${itemName} from shopping list?`,
+    );
+    if (deleteItem === false) return;
+    const deleteItemIndex = shoppingList[0].items.findIndex((obj) => {
+      return obj.shoppingListItemName === itemName;
```

Comment:
Did not know much about `findIndex` until now. Learned about this from you now. Thanks:)